mappings/net/minecraft/world/gen/Density.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_ojoovffl net/minecraft/world/gen/Density
+	FIELD f_mpylalvy SURFACE D
+	FIELD f_oaliifne UNRECOVERABLY_DENSE D
```

Comment:
Is `UNRECOVERABLY` a word? Surely something along the lines of `UNRECOVERABLE_DENSITY_LEVEL(S)` would be better in terms of coherent English. Not sure about the length though

Reply:
language is constantly evolving ;)